OpenSim/Examples/Plugins/VolumetricContactExample/VolumetricContactForce.h

```diff
@@ -0,0 +1,134 @@
+#ifndef _OPENSIM_VOLUMETRIC_CONTACT_FORCE_PLUGIN_H_
+#define _OPENSIM_VOLUMETRIC_CONTACT_FORCE_PLUGIN_H_
+/* -------------------------------------------------------------------------- *
+ *                        VolumetricContactForce.h                            *
```

Comment:
`VolumetricContactForce` might be a good base name for a family of elements especially if they will share the same (subset of) properties and sockets. Perhaps this class be called something like `EllipsoidHalfspaceVolumetricForce'?